go.mod

```diff
@@ -4,6 +4,7 @@ go 1.17
 
 require (
 	github.com/NVIDIA/go-nvml v0.11.6-0.0.20220823120812-7e2082095e82
+	github.com/NVIDIA/gpu-monitoring-tools v0.0.0-20211102125545-5a2c58442e48
```

Comment:
Deprecated?

Reply:
+1.  And we already depend on github.com/NVIDIA/go-nvml

Reply:
fixed